packages/manager/apps/web/client/app/hosting/database/order/components/private-database/translations/Messages_de_DE.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
`git mv` can be used to avoid all translations to be shown as new change.

Reply:
I used git move. Some times it shown as new file.